### PR TITLE
Detect public properties used via Subclass

### PR DESCRIPTION
**Pull request by staabm**
Utilize the declaring class, like we already do in ClassConstFetchCollector

### CODE REVIEW

File: tests/Rules/UnusedPublicPropertyRule/Fixture/StaticUsedInUnionA.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Rules\UnusedPublicPropertyRule\Fixture;
+namespace TomasVotruba\UnusedPublic\Tests\Rules\UnusedPublicPropertyRule\Fixture;
```
Comment (staabm) [inline tests/Rules/UnusedPublicPropertyRule/Fixture/StaticUsedInUnionA.php:5]: these pre-existing tests had wrong namespaces and therefore autolading did not work

**Comment by staabm** [comment]
should be good to go

**Comment by staabm** [comment]
Ping @TomasVotruba

**Review by TomasVotruba — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Looks good, thank you :+1:

**Comment by staabm** [comment]
thanks. would be great to get this released ;-)

**Pull request merged**